1906-09-17.xml

```diff
@@ -1441,16 +1439,4536 @@
+                            A. P. Mack</persName>, and agreed to. </p>
+                    <p>On the proposition of <persName>Mr. C. H. Rigg</persName>, seconded by
+                            <persName>Mr. Veasey</persName>, Messrs. A<persName>nnan Kirby
+                            Dexter</persName> and Co. were reappointed auditors of the company. </p>
+                    <p>In acknowledging a vote of thanks, the Chair- man said the duty which he had
+                        had to perform at that meeting had not been very arduous, and he hoped that
+                        if he had the pleasure of meeting the shareholders in future similar
+                        unanimity would be shown. </p>
+                    <p>The proceedings then terminated. </p>
+                </div>
+                <div type="section" feature="khedive">
+                    <head>THE KHEDIVE.s Yacht </head>
+                    <p>An interesting insurance was effected at Lloyd’s last week on the Khedive's
+                        pleasure yacht <name>Mahroussa</name>. This vessel, now valued at £100,000,
+                        has, as we recently reported, been entirely reconstructed, having been
+                        fitted on the Clyde with three of the most modern steam turbines and
+                        otherwise converted from a paddle into a screw steamer. The previous
+                        Thursday she succeessfully ran her official trial trip in the Firth of Clyde
+                        and attained a speed over a measured distance of 17.75 knots. The present
+                        insurance covers the yacht against all maritime perils on her voyage out to
+                            <placeName>Alexandria</placeName>, via <placeName>Gibraltar</placeName>
+                        and <placeName>Constantinople</placeName>, at which latter port it is
+                        anticipated that <persName>the Khedive</persName> will go on board. </p>
+                    <p>Until the vessel is officially delivered she is in the hands of the
+                        contractors, who have provided in the insurance contract that the yacht will
+                        be navigated as far as <placeName>Gibraltar</placeName> in the charge of a
+                        special pilot. Seeing that <name> the Mabroussa</name> was originally built
+                        of iron on the <placeName>Thames</placeName> as long ago as 1865, some
+                        underwriters have been surprised that she was worth the enormous amount
+                        spent upon her reconstruction, but they are unanimous in considering her a
+                        first-class risk for the voyage out, as she is practically an entirely new
+                        yacht, inferior probably to no other pleasure craft of the same type afloat.
+                            <name>The Mabroussa</name> is not the first yacht built for a
+        
```

Comment:
I am also confused to see wires of 1906-09-22 to show up in a 1906-09-17's newspaper. Is this the mistake of the newspaper itself?